src/content/pages/attend/_volunteer-closed.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Volunteering
+order: 20
+---
+
+import Button from "../../../components/Button.astro"
+
+// TODO: REMOVE `-closed` SUFFIX BEFORE PUBLISHING
+
+# Volunteer applications are now closed!
+
+Thank you all so much for the massive interest in volunteering for PyCon AU 2024! 
+We have closed our applications for volunteers for this year. If you have previously applied, we will contact you via the email you provided on if your application has been accepted or rejected. Given the limited amount of volunteer places, we unfortunately cannot guarantee your application will be accepted.
+
+
+
+## I missed the opportunity to volunteer. How else can I help? 
+
+### Become a session chair closer to the event!
+
+Session chairs are an integral part of the conference experience, as they are responsible for introducing and sending off each speaker during the main conference. 
+We will send more information to ticket holders about becoming a session chair in an email to the address you provided, within the weeks coming up to the event. 
+
+### Know someone who would make a good sponsor? Contact us!
+
+We're still looking for companies who want to support this event! We especially want to get in contact with Australian small businesses and community sponsors who want to give back to the local python community. If that sounds like you, or you know of a company that would be interested, email us at [sponsor@pycon.org.au](mailto:sponsor@pycon.org.au).
+See our [sponsors page](/sponsor/) for more details. 
```

Comment:
@devjack let me know if there's anything different you would like to place here.

Reply:
(Jack confirmed over discord that it looked good.)